packages/common-integration-tests/src/storage/storage.ts

```diff
@@ -71,169 +70,26 @@ export function runStorageServiceTests(
-          key,
-          intValue
-        );
-        switch (putIntResponse.type) {
-          case StoragePutResponse.Success: {
-            break;
-          }
-          case StoragePutResponse.Error: {
-            throw new Error(
-              `failed to put key: ${putIntResponse.message()} ${putIntResponse.toString()}`
-            );
-          }
-        }
-        const getIntResponse = await storageClient.get(testingStoreName, key);
-        expectWithMessage(() => {
-          expect(getIntResponse.type).toEqual(StorageGetResponse.Found);
-        }, `expected Found, received ${getIntResponse.toString()}`);
-        expect(getIntResponse.value()?.int()).toEqual(intValue);
-
-        // put/get a double value
-        const doubleValue = 42.42;
-        const putDoubleResponse = await storageClient.putDouble(
-          testingStoreName,
-          key,
-          doubleValue
-        );
-        switch (putDoubleResponse.type) {
-          case StoragePutResponse.Success: {
-            break;
-          }
-          case StoragePutResponse.Error: {
-            throw new Error(
-              `failed to put key: ${putDoubleResponse.message()} ${putDoubleResponse.toString()}`
-            );
-          }
-        }
-        const getDoubleResponse = await storageClient.get(
-          testingStoreName,
-          key
-        );
-        expectWithMessage(() => {
-          expect(getDoubleResponse.type).toEqual(StorageGetResponse.Found);
-        }, `expected Found, received ${getDoubleResponse.toString()}`);
-        expect(getDoubleResponse.value()?.double()).toEqual(doubleValue);
-
-        // put/get a string value
-        const stringValue = v4();
-        const putStringResponse = await storageClient.putString(
-          testingStoreName,
-          key,
-          stringValue
-        );
-        switch (putStringResponse.type) {
-          case StoragePutResponse.Success: {
-            break;
-          }
-          case StoragePutResponse.Error: {
-            throw new Error(
-              `failed to put key: ${putStringResponse.message()} ${putStringResponse.toString()}`
-            );
-          }
-        }
-        const getStringResponse = await storageClient.get(
-          testingStoreName,
-          key
-        );
-        expectWithMessage(() => {
-          expect(getStringResponse.type).toEqual(StorageGetResponse.Found);
-        }, `expected Found, received ${getStringResponse.toString()}`);
-        expect(getStringResponse.value()?.string()).toEqual(stringValue);
-
-        // put/get a bytes value
-        const bytesValue = new Uint8Array([1, 2, 3, 4]);
-        const putBytesResponse = await storageClient.putBytes(
-          testingStoreName,
-          key,
-          bytesValue
-        );
-        switch (putBytesResponse.type) {
-          case StoragePutResponse.Success: {
+      // WithStore creates a store then deletes it
+      await WithStore(storageClient, storeName, async () => {
```

Comment:
used WithStore in this test to make it more concise, but it does the same thing as before (calls create, create, delete)